quotes

```diff
@@ -1,4 +1,4 @@
-"If you want to be happy, be." - Leo Tolstoy
```

Comment:
Sorry! I should have just added a new quote at the bottom. Initially, I thought of making a new version with all my chosen quotes but then I stalled and decided to just try and watch how pull requests work.